__tests__/components/next_steps_test.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { NextSteps } from "../../components/next_steps";
+import { mount } from "enzyme";
+import configureStore from "redux-mock-store";
+import translate from "../fixtures/translate";
+
+const { axe, toHaveNoViolations } = require("jest-axe");
+expect.extend(toHaveNoViolations);
+
+describe("NextSteps", () => {
+  let props;
+  let mockStore, reduxState;
+  beforeEach(() => {
+    props = {
+      t: translate,
+      mapUrl: { query: {}, route: "/map" }
+    };
+
+    mockStore = configureStore();
+    reduxState = {};
+    props.store = mockStore(reduxState);
+    props.reduxState = reduxState;
```

Comment:
I think you can delete lines 19 and 21 and this should still work

Reply:
I think I removed them